ArchiSteamFarm/Commands.cs

```diff
@@ -0,0 +1,263 @@
+				return await func(bot, steamID, arguments).ConfigureAwait(false);
+			}
+
+			return null;
+		}
+
+		private static async Task<string> ResponseExit(Bot bot, ulong steamID, string[] args) {
+			if (steamID == 0) {
+				ASF.ArchiLogger.LogNullError(nameof(steamID));
+				return null;
+			}
+
+			if (!Bot.IsOwner(steamID)) {
+				return null;
+			}
+
+			Utilities.InBackground(
+				async () => {
+					await Task.Delay(1000).ConfigureAwait(false);
+					await Program.Exit().ConfigureAwait(false);
+				}
+			);
+
+			return FormatStaticResponse(Strings.Done);
+		}
+
+		private static async Task<string> ResponseSA(Bot bot, ulong steamID, string[] args) => await ResponseStatus(bot, steamID, new string[] { SharedInfo.ASF }).ConfigureAwait(false);
+
+		private static async Task<string> ResponseStatus(Bot bot, ulong steamID, string[] args) {
+			if (bot == null || steamID == 0 || args == null) {
+				ASF.ArchiLogger.LogNullError(nameof(bot) + " || " + nameof(steamID) + " || " + nameof(args));
+				return null;
+			}
+
+			if (args.Length == 0) {
+				return GetSingleBotStatusResponse(bot, steamID);
+			}
+
+			string botNames = Utilities.GetArgsAsText(args, 0, ",");
+			HashSet<Bot> bots = Bot.GetBots(botNames);
+			if (bots == null || bots.Count == 0) {
+				return Bot.IsOwner(steamID) ? FormatStaticResponse(string.Format(Strings.BotNotFound, botNames)) : null;
+			}
+
+			IEnumerable<(Bot Bot, Task<string> Task)> tasks = bots.Select(singleBot => (singleBot, Task.Run(() => GetSingleBotStatusResponse(singleBot, steamID))));
+			ICollection<(Bot Bot, string Response)> results;
+
+			switch (Program.GlobalConfig.OptimizationMode) {
+				case GlobalConfig.EOptimizationMode.MinMemoryUsage:
+					results = new List<(Bot Bot, string Response)>(bots.Count);
+					foreach ((Bot currentBot, Task<string> currentTask) in tasks) {
+						results.Add((currentBot, await currentTask.ConfigureAwait(false)));
+					}
+
+					break;
+				default:
+					results = await Task.WhenAll(tasks.Select(async task => (task.Bot, await task.Task.ConfigureAwait(false)))).ConfigureAwait(false);
+					break;
+			}
+
+			List<(Bot Bot, string Response)> validResults = new List<(Bot Bot, string Response)>(results.Where(result => !string.IsNullOrEmpty(result.Response)));
+			if (validResults.Count == 0) {
+				return null;
+			}
+
+			HashSet<Bot> botsRunning = validResults.Where(result => result.Bot.KeepRunning).Select(result => result.Bot).ToHashSet();
+
+			string extraResponse = string.Format(Strings.BotStatusOverview, botsRunning.Count, validResults.Count, botsRunning.Sum(bot => bot.CardsFarmer.GamesToFarm.Count), botsRunning.Sum(bot => bot.CardsFarmer.GamesToFarm.Sum(game => game.CardsRemaining)));
+			return string.Join(Environment.NewLine, validResults.Select(result => result.Response).Union(extraResponse.ToEnumerable()));
+		}
+
+		private static string GetSingleBotStatusResponse(Bot bot, ulong steamID) {
```

Comment:
Name it the same as parent function, so `ResponseStatus`. This one accepts less arguments anyway.